site/en/blog/new-in-extensions-2/index.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Extension Manifest Converter
+description: >
+  Open source tool to convert extensions to Manifest V3. You’ll still need to manually update any
+  service worker code that relies on a DOM, in addition to updating chrome.scripts.
+subhead: >
+  Easily convert an entire directory, extension zip file, or manifest.json file.
+date: 2021-04-19
+updated: 2021-04-19
+authors:
+  - solomonkinard
+  - dotproto
+tags:
+  - extensions
+draft: true
+---
+
+Extension Manifest Converter for Manifest V3
```

Comment:
There's some errant text here and I don't know why. Similarly below, there's "filenames" which aren't really spaced out right (they should be `// comments` in the code).